spec/hosts/roles/hexrot_spec.rb

```diff
@@ -32,16 +32,85 @@
           it { is_expected.to contain_class('docker') }
 
           %w[
+            anaconda
+            mate
             profile::core::common
             profile::core::debugutils
             profile::core::docker
             profile::core::docker::prune
             profile::core::ni_packages
```

Comment:
There also  isn't a class spec for this one. Please write a shared example for it.

Reply:
support/spec created for ni_packages 👍🏻